modules/serverless-kafka-sink-security-config.adoc

```diff
@@ -0,0 +1,103 @@
+// Module is included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sink-security-config_{context}"]
+= Configuring security for Kafka sinks
+
+You can configure security features, such as encryption and authentication, for Kafka sinks. You can do this by creating certificates as secrets, and then referencing these secrets from the `KafkaSink` object that you want to configure security for.
```

Comment:
I think this could be a bit misleading. The TLS and SASL configuration here is only for encryption and authentication that KafkaSink uses to connect to Kafka.

It doesn't configure security for the KafkaSink itself (anybody who can connect to the kafkasink's URL can still POST any cloudevents to the kafkasink)

Reply:
I think it is a valid concern perhaps the wording in https://github.com/openshift/openshift-docs/pull/50877/files seems concise enough and less misleading.

```
= Configuring SASL authentication for Kafka sinks

_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
```

Reply:
> (Any idea about a better wording or maybe explicitly say something about having the sink still open to anything?)

We can perhaps capture an issue for the open to anything problem and rework every addressable that we ship to mention the open to anything problem and how to address it

Reply:
Removing the paragraph
> You can configure security features, such as encryption and authentication, for Kafka sinks. You can do this by creating certificates as secrets, and then referencing these secrets from the `KafkaSink` object that you want to configure security for.

and leaving the part about SASL and TLS. Does this solve the issue?
@pierDipi can you open another issue please if you're saying that you think that's required and we'll need a follow up?